lisa/wlgen/rta.py

```diff
@@ -592,12 +610,38 @@ def __init__(self, start_pct=0, end_pct=100, delta_pct=10, time_s=1,
         # clip the last step
         steps[-1] = end_pct
 
+        # Clamp values for previous phases.
+        # These values are remembered so the clamp values are not set on the start
+        #   of every phase
```

Comment:
nit: you can leave just one space between # and the first word

Reply:
These comments have now been reformatted.